app/services/email_checker.rb

```diff
@@ -56,7 +56,8 @@ def compute_error
   # rubocop:enable Metrics/MethodLength
 
   def override_sendgrid?
-    @override_sendgrid
+    @override_sendgrid ||
+      ENV['ENABLE_SENDGRID_VALIDATIONS'].try(:downcase) != 'true'
```

Comment:
Spoke to @mloughran and agreed to move to Rails configuration to keep it consistent with the other environment variables.

Also the presence of the environment variable should be enough to enable the validations to keep consistency.

@tatyree I'll get working on that approach, let me know if that will still be confusing